go-runtime/compile/schema_test.go

```diff
@@ -281,9 +320,9 @@ func normaliseString(s string) string {
 }
 
 func TestErrorReporting(t *testing.T) {
-	if testing.Short() {
-		t.SkipNow()
-	}
+	// if testing.Short() {
```

Comment:
Should we go ahead and remove this block fully? I can't recall this test ever running slowly on my machine